lib/Installation/DiskActivation/AddZFCPDevicePage.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to the Add New ZFCP Device dialog
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::DiskActivation::AddZFCPDevicePage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+
```

Comment:
only one line between areas of code, also check at the end of the rest of the files between `}` and `1`.